lib/ui/src/containers/nav.js

```diff
@@ -95,6 +97,41 @@ const createMenu = memoize(1)((api, shortcutKeys, isFullscreen, showPanel, showN
   },
 ]);
 
+export const removeLeafStories = stories => {
+  return Object.values(stories).reduce((acc, item) => {
+    // filter out leaves
+    if (item.isLeaf) {
+      return acc;
+    }
+
+    const { children, ...rest } = item;
+    const component = { ...rest, isLeaf: true, isComponent: true, id: children[0] };
+    // make components leaves
```

Comment:
Can you expand this comment: "Make component leaves and link to the first story"